drivers/vhost/Kconfig

```diff
@@ -34,6 +34,13 @@ config VHOST_VSOCK
 	To compile this driver as a module, choose M here: the module will be called
 	vhost_vsock.
 
+config VHOST_SOF
+	bool "Vhost SOF driver"
+	default n
+	---help---
+	  SOF vhost VirtIO driver. It exports the same IPC interface, as the
+	  one, used for DSP communication, to Linux VirtIO guests.
+
```

Comment:
so this implements what, the front-end or the back-end ?

Reply:
all drivers under drivers/vhost are host side / back end